examples/allennlp/README.md

```diff
@@ -0,0 +1,6 @@
+# Hyperparameter optimization using Optuna
+
+This examples show two ways using Optuna for tuning AllenNLP models.
```

Comment:
A nit
```suggestion
These examples show two ways of using Optuna for tuning AllenNLP models.
```

Reply:
Thank you for correcting, I applied your suggestion. (a1abaa5)